i18n/gallery/es.json

```diff
@@ -21,10 +23,11 @@
   "I18N_GALLERY_CATEGORIES_GEOGRAPHY": "Geografía",
   "I18N_GALLERY_CATEGORIES_GOVERNMENT": "Gobierno",
   "I18N_GALLERY_CATEGORIES_HOBBIES": "Hobbies",
+  "I18N_GALLERY_CATEGORIES_INTERACTIVE_FICTION": "Ficción Interactiva",
```

Comment:
Me neither!! There is one exploration with this category between the ones that came with the code.